packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -2,6 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// reduced-test-set:
+//   This file is run as part of a reduced test set in CI on Mac and Windows
+//   machines.
+@Tags(<String>['reduced-test-set'])
+library;
```

Comment:
I've never seen this before. How do you know when to add this? 👀

Reply:
I think this is new, the analyzer complained that I didn't have this added in a file that used golden tests.

Reply:
Oh interesting. Thanks for the info!